File: C++/A1/070.cpp
#include <iostream>
#include <stdio.h>

using namespace std;

int main() {
    int N; // จำนวนเครื่อง
    if (!(cin >> N)) return 0;

    for (int i = 0; i < N; i++) {
        double plastic, can, glass; // ใช้ชื่อตัวแปรให้ตรงประเภทขยะ
        cin >> plastic >> can >> glass;

        double total = plastic + can + glass;
        printf("%.1f", total); // แสดงผลทศนิยม 1 ตำแหน่ง

        if (total > 50.0) cout << ",Overloaded"; // เกิน 50 กก.
        if (plastic > 20.0) cout << ",Check Type Plastic"; // เกิน 20 กก.
        if (can > 20.0) cout << ",Check Type Can";
        if (glass > 20.0) cout << ",Check Type Glass";
        
        cout << "\n";
    }
    return 0;
}

File: Python/A1/070.py
n = int(input())
for i in range(n):
    a,b,c = map(float,input().split())
    sum = a+b+c
    print(f"{sum:.1f}",end="")
    if(sum>50):print(",Overloaded",end="")
    if(a>20):print(",Check Type Plastic",end="")
    if(b>20):print(",Check Type Can",end="")
    if(c>20):print(",Check Type Glass",end="")
    print("")